src/ome_zarr_models/v04/multiscales.py

```diff
@@ -60,10 +256,69 @@ class MultiscaleGroupAttrs(Base):
     See https://ngff.openmicroscopy.org/0.4/#image-layout.
     """
 
-    multiscales: list[Multiscale] = Field(
+    multiscales: tuple[Multiscale, ...] = Field(
         ...,
         description="The multiscale datasets for this image",
         min_length=1,
     )
     omero: Omero | None = None
-    _check_unique = field_validator("multiscales")(_unique_items_validator)
+
+
+class MultiscaleGroup(GroupSpec[MultiscaleGroupAttrs, ArraySpec | GroupSpec]):
```

Comment:
How is this meant to be used? I am a bit confused/not understanding what the purpose of this class is.

Reply:
this class will model an actual zarr group, i.e. the attributes (which are JSON and ought to contain multiscales etc) and the arrays (which ought to be the ones named by the `path` attribute in `datasets`, and which ought to have shape that's consistent with the number of axes, etc)

Reply:
this class is also the thing we deserialize a zarr group into